compiler/passes/docs.cpp

```diff
@@ -295,6 +295,34 @@ std::string generateSphinxProject(std::string dirpath) {
   return std::string(moddir);
 }
 
+/* Checks for the valid Project Version (non-recursively). If an error occurs, exit and report
+ * error.
+ */
+char * checkProjectVersion(char * projectVersion) {
+  bool check = true;
+  const char * tags[] = {"alpha", "beta", "rc"};
+  for(int i=0; i<5; i++) {
+    if(i%2 != 0 && projectVersion[i] != '.') {
+      check = false;
+      break;
+    } else if(i%2 == 0 && ((int)projectVersion[i] > (int)'9' || (int)projectVersion[i] < (int)'0')) {
+      check = false;
+      break;
+    }
```

Comment:
This assumes that versions will always be single digits, but that's not necessarily going to be the case.  For instance, you couldn't specify `1.20.0` as the documentation version.  You also couldn't provide something that dropped the minor version, e.g `21.3`.

Reply:
Changed it to accept also multi digits